specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/ServiceEndpointPolicyDefinitionGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters" : {
+    "api-version": "2018-07-01",
+    "subscriptionId" : "subid",
+    "resourceGroupName" : "rg1",
+    "networkSecurityGroupName" : "testnsg",
+    "securityRuleName": "rule1"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "name": "rule1",
+        "id":"/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityGroups/testnsg/securityRules/rule1",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "protocol": "*",
+          "sourcePortRange": "*",
+          "destinationPortRange": "80",
+          "sourceAddressPrefix": "*",
+          "destinationAddressPrefix": "*",
+          "access": "Allow",
+          "priority": 130,
+          "direction": "Inbound"
```

Comment:
Additional properties not allowed:
    direction,priority,access,destinationAddressPrefix,sourceAddressPrefix,destinationPortRange,sourcePortRange,protocol are not defined in `ServiceEndpointPolicyDefinition`